plugins/inputs/opcua/sample.conf

```
@@ -123,6 +123,11 @@
   #   identifier_type = ""
   #   identifier = ""
 
+  ## Include additional Fields in each metric
+  # [inputs.opcua.additional_fields]
+    ## OPC-UA DataType
+    # include_datatype = true
```

The commented out value should represent the default value
```suggestion
    # include_datatype = false
```

---

ack